mirage/seed_image/catalog_seed_image.py

```diff
@@ -1877,6 +1883,59 @@ def readPointSourceFile(self, filename):
 
         return gtab, pflag, msys
 
+    def select_magnitude_column(self, catalog, catalog_file_name):
+        """Select the appropriate column to use for source magnitudes from the input source catalog. If there
+        is a specific column name constructed as <instrument>_<filter>_magnitude to use for source magnitudes
+        (e.g. nircam_f200w_magnitude) then use that. (NOTE: for FGS we search for a column name of
+        'fgs_magnitude'). If not, check for a generic 'magnitude' column. If neither are present, raise an
+        error.
+
+        Parameters
+        ----------
+
+        catalog : astropy.table.Table
+            Source catalog
+
+        catalog_file_name : str
+            Name of the catalog file. Used only when raising errors.
+
+        Returns
+        -------
+
+        specific_mag_col : str
+            The name of the catalog column to use for source magnitudes
+        """
+        # Determine the filter name to look for
+        if self.params['Inst']['instrument'].lower() in [nircam, niriss]:
```

Comment:
This line should be: 
`if self.params['Inst']['instrument'].lower() in ['nircam', 'niriss']:`
so the instrument names are strings. I got an error when running this because it didn't know what `nircam` was 😬

Reply:
I'm really confused as to why this didn't fail the CI....

Reply:
Or in my tests...
Very strange. Good catch.

Reply:
Quotes added.